examples/docking-protein-DNA/docking-protein-DNA-cltsel-full.cfg

```diff
@@ -43,9 +43,9 @@ threshold=4
 
 [seletopclusts]
 ## select the best 4 clusters
-# top_cluster = [1, 2, 3, 4]
+# top_cluster = 4
```

Comment:
This line is commented out. But does it mean that if you uncomment it the top4 best clusters will be selected? I.e. no longer needed to specify them explicitely?

Reply:
exactly! the point of this issue was to make the selection easier for the user when the number of clusters they want to select is high